frontend/test/metabase/scenarios/models/reproductions/20963-can-not-convert-question-with-snippets-to-model.cy.spec.js

```diff
@@ -0,0 +1,60 @@
+import { restore, modal } from "__support__/e2e/cypress";
+import {
+  openDetailsSidebar,
+  getDetailsSidebarActions,
+} from "../helpers/e2e-models-helpers";
+
+const snippetName = `string 'test'`;
+const questionName = "Converting questions with snippets to models";
+
+describe("issue 20963", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should allow converting questions with static snippets to models (metabase#20963)", () => {
+    cy.visit("/");
+
+    cy.findByText("New").click();
+    cy.findByText("SQL query").click();
```

Comment:
You can also use [`openNativeEditor` e2e helper](https://github.com/metabase/metabase/blob/7f9aa58ae1e0fb09115fa39505664457617c54fc/frontend/test/__support__/e2e/helpers/e2e-misc-helpers.js#L27). It will also select the ACE editor and give it an alias, so you can just do `cy.get("@editor").type("...")`

Reply:
That sounds neat. I'll try that.

Reply:
I modified the PR to use your suggestion 🙏